runtimes/web/src/apu-worklet.ts

```diff
@@ -126,6 +129,7 @@ class APUProcessor extends AudioWorkletProcessor {
 
         const channelIdx = flags & 0x3;
         const mode = (flags >> 2) & 0x3;
+        const pan = ((flags >> 4) & 0x3) % 3;
```

Comment:
Is this the same as:

```suggestion
        const pan = (flags >> 4) & 0x3;
```